app/data/tasks/deduplicate_records.py

```diff
@@ -0,0 +1,185 @@
+from django.conf import settings
+from django.core.paginator import Paginator
+
+from celery import shared_task
+from celery.utils.log import get_task_logger
+
+from data.models import (DedupeJob, RecordDuplicate)
+from ashlar.models import Record
+
+import datetime
+from pytz import timezone
+
+logger = get_task_logger(__name__)
+
+
+def get_time_allowance():
```

Comment:
Would it work to make these default parameters to the job itself; then they could be overridden if we wanted to call the task manually? Something like 
```python
def find_duplicate_records(time_allowance=None, distance_allowance=None):
    if distance_allowance is None:
        distance_allowance = settings.DEDUPE_DISTANCE_DEGREES
```